pages/about/groups/task-forces/pdf-aam.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Portable Document Format Accessibility APIs Mapping (PDF-AAM) Task Force
```

Comment:
Ideally, the <abbrev> element for 'APIs' should be on the first instance of the page.

Reply:
It is not possible to use `abbr` according to the HTML specification. If we were to expand the abbreviation in plain text, such as 'Accessibility Programming Interfaces (APIs)" this would make for quite a long title text. I have added now `abbr` elements in the first paragraph of these two pages.

Reply:
Yes, I agree, expanding the abbreviation in this heading would hurt readability.

I'm confused about the interpretation of the HTML Standard, though.

The HTML spec says that `abbr` can be used in phrasing content. 
This, as I understand it, includes phrasing content in headings. 

The following example from w3.org has 2 paragraphs separated by a heading. The heading is described as containing 'phrasing content that is not a paragraph':
https://www.w3.org/TR/2011/WD-html5-author-20110809/content-models.html#paragraphs

I've seen `abbr` implemented not only in headings, but also breadcrumbs and navigation elements.
Don't let this stop publication, but I'd like to check this.

Reply:
Yes, it is totally fine to use `abbr` on headings. My interpretation is that it is not allowed to use it within the [title element](https://html.spec.whatwg.org/multipage/semantics.html#the-title-element), because the title element only allows for text content.

72287c726f7d7d2342d11626a0a4579752cbadf3 updates this. Let me know what you think about this.

Reply:
I've added a comment to [72287c7](https://github.com/w3c/wai-website/commit/72287c726f7d7d2342d11626a0a4579752cbadf3)

Reply:
Thanks. I've taken a pass at this in the two pages. Now abbr elements are on h1, they've been removed from the summary paragraphs, and abbreviations are expanded in the second sentence of the summary.